homeassistant/components/binary_sensor/zwave.py

```diff
@@ -80,26 +87,28 @@ def __init__(self, value, device_class, hass, re_arm_sec=60):
         self.invalidate_after = dt_util.utcnow() + datetime.timedelta(
             seconds=self.re_arm_sec)
         # If it's active make sure that we set the timeout tracker
-        if value.data:
+        if self._state:
```

Comment:
We can remove this whole if-block.

We only need a timeout to reset the value to `False` again if it was `True`. Since it's not `True` here, we should be good.